docs/src/main/asciidoc/management-interface-reference.adoc

```diff
@@ -27,10 +27,12 @@ quarkus.management.enabled=true
 ----
 
 By default, management endpoints will be exposed on: `http://0.0.0.0:9000/q`.
-For example, `http://0.0.0.0:9000/q/health/ready` for the readiness probe.
+For example, if you have `smallrye-health` installed, the readiness probe will be exposed at `http://0.0.0.0:9000/q/health/ready`.
 
 SmallRye Health Checks, SmallRye Metrics, Micrometer and Info endpoints will be declared as management endpoints when the management interface is enabled.
 
+NOTE: The management interface will not work properly without extensions that use it. e.g. `smallrye-health`, `quarkus-smallrye-openapi`.
```

Comment:
It's not that it does not work. It's on purpose. If there no endpoint exposed, it is disabled.

Reply:
Thanks for clarifying, I've updated the phrasing. Let me know if it makes sense.